src/pysoem/pysoem.pyx

```diff
@@ -877,6 +918,8 @@ cdef class CdefSlave:
         if self._ecx_contextt == NULL:
             raise UnboundLocalError()
```

Comment:
And here: This two lines could actually be removed, right? As `__check_context_is_initialized` is doing the same?